src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationBinderTests.cs

```diff
@@ -112,6 +112,75 @@ public class DerivedOptionsWithIConfigurationSection : DerivedOptions
             public IConfigurationSection DerivedSection { get; set; }
         }
 
+#if NETCOREAPP        
```

Comment:
We do it in other tests. For example:

https://github.com/dotnet/runtime/blob/d2688883510318aa0114268930fb2022d3dc64cf/src/libraries/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj#L14

https://github.com/dotnet/runtime/blob/d2688883510318aa0114268930fb2022d3dc64cf/src/libraries/Microsoft.VisualBasic.Core/tests/Microsoft.VisualBasic.Core.Tests.csproj#L7

I think it is better to ensure this works on .NET Framework, since the Extensions libraries support it.